ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_namespace_functions.py

```diff
@@ -343,3 +343,81 @@ def test_jax_numpy_reshape(
         a=x[0],
         newshape=shape,
     )
+
+
+@st.composite
+def _arrays_idx_n_dtypes(draw):
+    num_dims = draw(st.shared(helpers.ints(min_value=1, max_value=4), key="num_dims"))
+    num_arrays = draw(
+        st.shared(helpers.ints(min_value=2, max_value=4), key="num_arrays")
+    )
+    common_shape = draw(
+        helpers.lists(
+            arg=helpers.ints(min_value=2, max_value=3),
+            min_size=num_dims - 1,
+            max_size=num_dims - 1,
+        )
+    )
+    unique_idx = draw(helpers.ints(min_value=0, max_value=num_dims - 1))
+    unique_dims = draw(
+        helpers.lists(
+            arg=helpers.ints(min_value=2, max_value=3),
+            min_size=num_arrays,
+            max_size=num_arrays,
+        )
+    )
+    xs = list()
+    input_dtypes = draw(
+        helpers.array_dtypes(available_dtypes=draw(helpers.get_dtypes("valid")))
+    )
+    for ud, dt in zip(unique_dims, input_dtypes):
+        x = draw(
+            helpers.array_values(
+                shape=common_shape[:unique_idx] + [ud] + common_shape[unique_idx:],
+                dtype=dt,
+            )
+        )
+        xs.append(x)
+    return xs, input_dtypes, unique_idx
+
+
+@st.composite
+def _dtype_n_with_out(draw):
+    dtype = draw(helpers.get_dtypes("float", none=True))
```

Comment:
is it possible to test the larger set of "valid" or "numeric" type?

Reply:
Hi, not sure how to go about this, something like this? 
`dtype = draw(helpers.get_dtypes("float", none=True), helpers.get_dtypes("valid", none=True))`

Reply:
hey! sorry for the late response, you can directly use `dtype=helpers.get_dtypes("numeric", none=True, full=False)` in the `@given` data generation if you're generating a single dtype! 

Here's a more detailed explanation of the helper functions in the docs - https://lets-unify.ai/ivy/deep_dive/17_ivy_frontends_tests.html#hypothesis-helpers